app/js/authentication_controller.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/* global User, Utils */
+
+(function(exports) {
+  function update_error_list(list, errors) {
```

Comment:
I'm not fond of update in the name since the first action empties the list. If I call this method twice with the same list, only the errors from the second call will show up. The name should reflect that. I don't have a good name suggestion though.